tools/generator/templates/source.go.tmpl

```diff
@@ -0,0 +1,25 @@
+// Package {{.PackageName}} provides details for the {{.Name}} applet.
+package {{.PackageName}}
+
+import (
+	_ "embed"
+
+	"tidbyt.dev/community/apps/manifest"
```

Comment:
Note: This import will need to change, given we are migrating the manifest to the `pixlet` repo. I don't know when it makes sense to do that change, but it will require updates to our internal repo and the community repo.